src/functionalTest/java/uk/gov/hmcts/sptribs/wa/WATaskRegisterNewCaseFT.java

```diff
@@ -40,7 +42,7 @@ public class WATaskRegisterNewCaseFT extends FunctionalTestSuite {
     public void should() throws IOException, InterruptedException {
         String newCaseId = String.valueOf(createAndSubmitTestCaseAndGetCaseReference());
 
-        System.out.println(newCaseId);
+        log.debug(newCaseId);
```

Comment:
Do we want to add any additional info to this log statement? eg. "New case created: " + newCaseId?